src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -199,19 +218,19 @@ const PastAnalysisTable = () => {
           parent: indexInRows,
           fullWidth: true,
           cells: [{
-            title: (<SeriesTable studyInstance={analysis} isProcessing={isProcessing}></SeriesTable>)
+            title: (<SeriesTable studyInstance={analysis} isProcessing={isProcessing} classifications={classifications}></SeriesTable>)
           }]
         })
```

Comment:
Can you please do me a favor and add semicolons after `rows.push();`. Lines 214 and 223. Thanks